src/containers/Home/PostForm.tsx

```diff
@@ -15,22 +30,42 @@ export default () => {
     ev.preventDefault()
     textareaRef.current!.focus()
   })
-
   const [draft, setDraft] = useState('')
-  const [draftDisabled, setDraftDisabled] = useState(false)
   const submitDraft = async () => {
     setDraftDisabled(true)
-    if (draft.trim().length > 0) {
+    if (draft.trim().length > 0 || 1 <= files.length) {
       try {
-        await seaClient.post('/v1/posts', { text: draft })
+        await seaClient.post('/v1/posts', {
+          text: draft,
+          fileIds: files.map(file => file.id)
+        })
         setDraft('')
+        setFiles([])
       } catch (e) {
         // TODO: Add error reporting
         console.error(e)
       }
     }
     setDraftDisabled(false)
   }
+  const onFileSelect = async (event: React.ChangeEvent<HTMLInputElement>) => {
+    event.preventDefault()
+    if (event.target.files != null) {
+      await Array.from(event.target.files).map(
+        async file => await uploadAlbumFile(file)
+      )
+    }
+  }
+  const onPaste = async (event: React.ClipboardEvent) => {
+    if (!draftDisabled) {
```

Comment:
return early してください。見通しが良くなります

```suggestion
    if (draftDisabled) return
```